packages/karma-esm/demo/import-map/karma.config.js

```diff
@@ -19,7 +19,11 @@ module.exports = config => {
     customLaunchers: {
       ChromeHeadlessNoSandbox: {
         base: 'ChromeHeadless',
-        flags: ['--no-sandbox', '--disable-setuid-sandbox'],
+        flags: [
+          '--no-sandbox',
+          '--disable-setuid-sandbox',
+          '--enable-experimental-web-platform-features',
```

Comment:
I dont think we should enable this by default for everyone, because it enabled more than just this.

Doesnt karma reuse your existing chrome indtallation? Otherwise, we should use document this step for users in their own config.

Reply:
> I dont think we should enable this by default for everyone, because it enables more than just import maps.

Yes this should be opt in, documented together importMap option

> Doesnt karma reuse your existing chrome indtallation?

No. I set the flag in Chrome but when running in karma seems all flags are reset

Reply:
Added documentation